lib/src/executable/compile_stylesheet.dart

```diff
@@ -79,6 +79,7 @@ Future compileStylesheet(ExecutableOptions options, StylesheetGraph graph,
   if (options.color) buffer.write('\u001b[32m');
 
   var sourceName = source == null ? 'stdin' : p.prettyUri(p.toUri(source));
+  destination = p.prettyUri(destination);
```

Comment:
Rather than modifying this variable, can you either create a new `destinationVariable` or include the call to `prettyUri()` inline in the `buffer.write()` call?

Reply:
Would something like `var prettyDestination = p.prettyUri(destination);`
and then replacing `$destination` with `$prettyDestination` work?

Reply:
I would prefer `destinationName`, since that matches `sourceName`, but yes.

Reply:
Thanks, will do!

Reply:
Done!